view/business-process-official.js

```diff
@@ -11,11 +11,17 @@ exports._match = 'businessProcess';
 exports['sub-main'] = {
 	class: { content: true, 'user-forms': true },
 	content: function () {
+		var hasDocuments = or(
+			this.businessProcess.requirementUploads.applicable._size,
+			this.businessProcess.paymentReceiptUploads.applicable._size,
+			this.businessProcess.certificates.userApplicable._size
```

Comment:
I'm not sure that we should include `userApplicable` collection for certificates in context of official. I think that for official it should be `applicable`.